scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -325,7 +327,7 @@ See the [OSCA chapter on Using the corrected values](https://bioconductor.org/bo
 Before we can compare the gene expression profiles of myoblasts in ARMS vs. ERMS samples, we will need to "pseudo-bulk" the gene counts. 
 Pseudo-bulking creates a new counts matrix that contains the sum of the counts from all cells with a given label (e.g., cell type) for each sample ([Tung _et al._, 2017](https://doi.org/10.1038/srep39921)). 
 If we were to keep each cell's counts separate, they would be treated as replicates, leading to inflated statistics. 
-By pseudo-bulking first, will now have one count for each gene for each sample and we can take advantage of well-established methods for differential expression with bulk RNA-seq.
+By pseudo-bulking first, we will now have one count for each gene for each sample and we can take advantage of well-established methods for differential expression with bulk RNA-seq.
 
 Pseudo-bulking is implemented prior to differential expression analysis on single-cell data for the following reasons: 
```

Comment:
I'd rephrase this slightly to match with bullets - 
```suggestion
Pseudo-bulking is implemented prior to differential expression analysis on single-cell data because it: 
```